pages/content/amp-dev/documentation/guides-and-tutorials/integrate/embed-stories-nonamp.md

```diff
@@ -0,0 +1,542 @@
+
+```html
+<body>
+...
+  <amp-story-player style="width: 370px; height: 622px;">
+...
+  </amp-story-player>
+...
+</body>
+```
+
+We recommend maintaining a 69:116 aspect ratio for the best user experience, but you may define any width and height.
+
+
+### Responsive sizing
+
+The story player's responsiveness works as any other block element. Use CSS to maintain width and height ratios, such as the example below:
+html
+```
+<amp-story-player style="width: 50vw; height: 84.05vw;">
+  ...
+</amp-story-player>
+```
+
+## Configure the Web Story Player behavior
+
+The Web Story Player allows you to specify behavior, such as autoplaying stories or circular wrapping. These are specified in the Web Story JSON behavior configuration. 
+
+```html
+<amp-story-player style="width: 370px; height: 622px;">
+<script type="application/json">
+    {
+      "behavior": {
+        "autoplay": false
+      }
+    }
+  </script>
+...
+</amp-story-player>
+```
+
+# Display a Web Story 
+
+Link to a Web Story by including an `<a>` tag with the `href` attribute pointed to the desired Web Story URL within the `<amp-story-player>` element. The `href` endpoint may be the URL of a hosted Web Story or a relative path. Place the title of the story within the `<a>` tags.
+
+[example preview="top-frame" playground="true"]
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <script
+      async
+      src="https://cdn.ampproject.org/amp-story-player-v0.js"
+    ></script>
+    <link
+      href="https://cdn.ampproject.org/amp-story-player-v0.css"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <style>
+      header {
+        height: 8vh;
+        color: #545454;
+        background-color: #DDB556;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      }
+      amp-story-player {
+        margin: 1rem auto;
+      }
+    </style>
+  </head>
+  <body>
+    <header>
+      <h1>
+        Page Header
+      </h1>
+    </header>
+    <h1>
+      Article Title
+    </h1>
+    <amp-story-player style="width: 370px; height: 622px;">
+      <a href="https://ampfest-story-player-demo.web.app/examples/amp-story/AMPFestPlayerDemo/s3"></a>
+    </amp-story-player>
+  </body>
+</html>
+```
+[/example]
+
+
+The Web Story Player displays the cover page of the embedded story. It autoplays the story when the viewer becomes visible in the user's viewport. You may opt out of autoplay by specifying `"autoplay": false` in the Web Story Player behavior configuration. 
+
+
+## Provide a placeholder
+
+Include a representative poster image by adding an <img> tag as a child of the story's <a> tag with the following configuration. The AMP story player displays this image while loading the full story.
+
+
+```html
+<amp-story-player style="width: 50vw; height: 83.35vw;">
+  <a href="https://www.example.com/story.html">
+    <img src="https://www.example.com/assets/cover1.html" loading="lazy" width="100%" height="100%" amp-story-player-poster-img>
```

Comment:
The validation rules expect `data-amp-story-player-poster-img` with a `data-` prefix.
@Enriqe what is this attribute for? Why is it mandatory? I see 0 references to it in our CSS or JS, do we use it?

Reply:
Yep, good catch. This should be `data-amp-story-player-poster-img`, we validate this in the AMP version but don't do anything with it just yet.

For context, Jon [recommended](https://github.com/ampproject/amphtml/issues/30512#issuecomment-703995772) to include this attribute for future-proofing.